From bcd276b5b62e1b5ff46eadad71417de076797ec8 Mon Sep 17 00:00:00 2001
From: lujingxiao <lujingxiao@huawei.com>
Date: Tue, 22 Jan 2019 17:26:35 +0800
Subject: [PATCH 078/111] spec:  add missing sysconfigs to rpm

reason: add missing sysconfigs to rpm, including
- /etc/sysconfig/docker
- /etc/sysconfig/docker-network
- /etc/sysconfig/docker-storage

Partly cherry-pick from 17.06 780e079a

Change-Id: I7a31bc6d99bab6c6a7a60988bc66db2d48f6ca4b
Signed-off-by: Lei Jitang <leijitang@huawei.com>
Signed-off-by: lujingxiao <lujingxiao@huawei.com>
---
 .../contrib/init/sysvinit-redhat/docker-network    |  2 ++
 .../contrib/init/sysvinit-redhat/docker-storage    | 14 ++++++++++++++
 5 files changed, 27 insertions(+), 4 deletions(-)
 create mode 100644 components/engine/contrib/init/sysvinit-redhat/docker-network
 create mode 100644 components/engine/contrib/init/sysvinit-redhat/docker-storage

diff --git a/components/engine/contrib/init/sysvinit-redhat/docker-network b/components/engine/contrib/init/sysvinit-redhat/docker-network
new file mode 100644
index 0000000000..048d1582ec
--- /dev/null
+++ b/components/engine/contrib/init/sysvinit-redhat/docker-network
@@ -0,0 +1,2 @@
+# /etc/sysconfig/docker-network
+DOCKER_NETWORK_OPTIONS=
diff --git a/components/engine/contrib/init/sysvinit-redhat/docker-storage b/components/engine/contrib/init/sysvinit-redhat/docker-storage
new file mode 100644
index 0000000000..3dc16542ff
--- /dev/null
+++ b/components/engine/contrib/init/sysvinit-redhat/docker-storage
@@ -0,0 +1,14 @@
+# This file may be automatically generated by an installation program.
+
+# By default, Docker uses a loopback-mounted sparse file in
+# /var/lib/docker.  The loopback makes it slower, and there are some
+# restrictive defaults, such as 100GB max storage.
+
+# If your installation did not set a custom storage for Docker, you
+# may do it below.
+
+# Example: Use a custom pair of raw logical volumes (one for metadata,
+# one for data).
+# DOCKER_STORAGE_OPTIONS = --storage-opt dm.metadatadev=/dev/mylogvol/my-docker-metadata --storage-opt dm.datadev=/dev/mylogvol/my-docker-data
+
+DOCKER_STORAGE_OPTIONS=
-- 
2.17.1

